test/models/templateMeta.test.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const { assert } = require('chai');
+const { models } = require('../..');
+const { validate } = require('../helper');
+
+describe('model template', () => {
+    describe('base', () => {
+        it('validates the base', () => {
+            assert.isNull(validate('pipelineTemplate.yaml', models.templateMeta.base).error);
+        });
+    });
+
+    describe('create', () => {
+        it('validates the create', () => {
+            assert.isNull(validate('pipelineTemplate.create.yaml', models.templateMeta.create).error);
+        });
+
+        it('fails the create', () => {
+            assert.isNotNull(validate('empty.yaml', models.templateMeta.create).error);
+        });
+    });
+
+    describe('get', () => {
+        it('validates the get', () => {
```

Comment:
Lets cover below success scenarios
- includes all fields
- without optional fields

Reply:
We need to add test to cover # 2